subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -102,6 +82,17 @@ config BT_FAST_PAIR_GATT_SERVICE
 
 if BT_FAST_PAIR_GATT_SERVICE
 
+config BT_FAST_PAIR_GATT_SERVICE_MODEL_ID
+	bool "Fast Pair Service's Model ID GATT characteristic"
```

Comment:
Maybe we could use conditional prompt here and allow user to configure this properly only for unknown use-case: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#optional-prompts? That would greatly reduce odds of breaking configuration accidentally.

Sorry that I overlooked this possibility during previous round of review.